subsys/bluetooth/mesh/Kconfig.models

```diff
@@ -289,6 +289,17 @@ config BT_MESH_LIGHT_CTRL_REG_SPEC_INTERVAL
 	  Update interval of the specification-defined illuminance regulator (in milliseconds).
 
 endif #BT_MESH_LIGHT_CTRL_REG_SPEC
+
+config BT_MESH_LIGHT_CTRL_AMB_LIGHT_LEVEL_TIMEOUT
+	int "Ambient light level report timeout"
```

Comment:
Specify unit for this somewhere? Not obvious at all now that it is seconds. Also, a 5 second default sounds like it's very short?